sklearn/externals/funcsigs.py

```diff
@@ -20,8 +20,8 @@
 __all__ = ['BoundArguments', 'Parameter', 'Signature', 'signature']
 
 
-_WrapperDescriptor = type(type.__call__)
```

Comment:
Is that a backport? I don't understand what's happening here (but I'm also not familiar with the code).